src/CMakeLists.txt

```diff
@@ -15,15 +26,20 @@ target_compile_definitions(journald
         -DG_LOG_DOMAIN="eventd-journald")
 target_link_libraries(journald
     PRIVATE
-        ${EventdEvent_LIBRARIES}
+        ${Eventd_LIBRARIES}
         ${EventdPlugin_LIBRARIES}
+        ${EVENTDHELPERS_LIBRARY}
         ${SystemdJournal_LIBRARIES})
 set_target_properties(journald
     PROPERTIES
         PREFIX    ""
         NO_SONAME 1)
 
+pkg_config_variable(libeventd-plugin pluginsdir EVENTD_PLUGINS_DIR_INTERNAL)
+set(EVENTD_PLUGINS_DIR ${EVENTD_PLUGINS_DIR_INTERNAL}
```

Comment:
It will install to the exact path the eventd found by pkg-config will search.
The path will be absolute, but pkg-config should respect everything you pass to it.
As a packager, it is a totally fine and desirable behaviour.

In Autotools, that actually means ignoring `--prefix` but still respect `DESTDIR` (used by package managers to put all files in a separate directory).

E.g. for cross-compiling, you set `PKG_CONFIG` to `<arch>-pkg-config` and it will search in `/usr/<arch>/lib/pkgconfig`. The found `libeventd-plugin.pc` file is likely to have `/usr/<arch>/lib/eventd/plugins` set as pluginsdir (unless this specific path was overridden at configure time).